docs/appendices/release-notes/4.1.0.rst

```diff
@@ -218,36 +221,35 @@ Observability improvements
 
 - Added a ``node`` column to :ref:`sys.jobs_log <sys-logs>`.
 
-- Statements containing limits, filters, window functions or table functions
+- Statements containing limits, filters, window functions, or table functions
   will now be labelled accordingly in :ref:`sys-jobs-metrics`.
 
 
 Others
 ------
 
 - Changed the default for :ref:`sql_ref_write_wait_for_active_shards` from
-  ``ALL`` to ``1``. This will improve the out of box experience as it allows a
-  subset of nodes to become unavailable without blocking write operations. See
-  the documentation for more details about the implications.
+  ``ALL`` to ``1``. This update allows a subset of nodes to become unavailable
+  without blocking write operations.
```

Comment:
Not sure what the improvement here is, I prefer the existing one.

Reply:
Basically it's shorter, and the "see the documentation" is too general. Either it should be specific (with a link to the relevant section, perhaps); or it's about the fact that one can always refer to the docs for more info, which is implicit in general and in which case it's not really necessary to mention it here.

Reply:
I should have been more verbose, sorry. I think the old "This will improve the out of box experience as it..." is worth keeping.

Reply:
I agree with Basti, I  think the old version is more useful. 

> Either it should be specific (with a link to the relevant section, perhaps)

The setting is linked, and it contains information about the implications.

Reply:
Okay, I have made a rewrite instead, that will make it clearer what is intended I think.